### PR TITLE
Prepare production release

### PR DESCRIPTION
**Pull request by willfish**
### What

- [x] Update dispatch workflow for production to match our new matrix rules and env var names
- [x] Fix slack notification urls
- [x] Remove reference to cds-test

### Why

We need all of these things to release to production

### A/C

- [x] !!!Excludes the current uk production AKA uk-old in the service matrix!!!
- [ ] Deployments to the new uk and xi services work (can only test after merging, sadly)

### CODE REVIEW

**Review by tavy88 — APPROVED** [review]
Paired so happy with it!

File: README.md [hunk 1]
```diff
@@ -68,15 +68,13 @@ In case of any errors, changes (per single update) are roll-backed and record it
...
 
-
 Set the following ENV variables:
 * CF_USER
 * CF_PASSWORD
 * CF_ORG
 * CF_SPACE
 * CF_APP
 * CF_APP_WORKER
-* HEALTHCHECK_URL
```
Comment (mafonso) [inline README.md:79]: Is this really not needed  for the API service?

Comment (willfish) [reply README.md:79]: Is dead

File: .github/workflows/deploy_production.yml [hunk 2]
```diff
@@ -2,10 +2,13 @@ name: "Production deployment"
 on: [workflow_dispatch]
 jobs:
   deploy_production:
-    if: github.ref == 'refs/heads/production_notyet'
+    if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 1
+      fail-fast: false
+      max-parallel: 3
```
Comment (willfish) [inline .github/workflows/deploy_production.yml:9]:
```suggestion
      max-parallel: 2
```

**Pull request merged**